alembic/versions/f960459f3067_change_parent_id_to_string_in_the_.py

```diff
@@ -0,0 +1,36 @@
+"""Change parent_id to String in the Structures Model
+
+Revision ID: f960459f3067
+Revises: 3942aff720bb
+Create Date: 2019-04-25 20:22:17.983064
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'f960459f3067'
+down_revision = '3942aff720bb'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('office_structures', 'parent_id',
+               existing_type=sa.INTEGER(),
+               type_=sa.VARCHAR(),
```

Comment:
In some migration files, the type is set to String. Here is it set to VARCHAR. What are the implications of this?

Reply:
It will still set the field to a `String` as is required.